azure_functions_worker/testutils_lc.py

```diff
@@ -181,7 +201,7 @@ def spawn_container(self,
         self._ports[self._uuid] = port_number
 
         # Wait for three seconds for the container to be in ready state
-        time.sleep(3)
+        time.sleep(6)
```

Comment:
also update in comment and why 6 seconds?

Reply:
Since we are mounting the functions library too, this was taking a bit more time. Keeping it at 5 sec failed. 6 sec was the sweet spot which got it working consistently